src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,74 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService mnfService = (ManufacturerService)
+                injector.getInstance(ManufacturerService.class);
+        Manufacturer manufacturerOne = new Manufacturer("Nissan", "Japan");
+        Manufacturer manufacturerTwo = new Manufacturer("Toyota", "Japan");
```

Comment:
```suggestion
        Manufacturer nissanManufacturer = new Manufacturer("Nissan", "Japan");
        Manufacturer toyotaManufacturer = new Manufacturer("Toyota", "Japan");
```

It looks better such way 🙂